frontend/src/components/countryLocator.svelte

```diff
@@ -0,0 +1,38 @@
+<script lang="ts">
+  import { onMount } from "svelte"
+  import { patron } from "../stores/patron"
+  import { currentCountry } from "../stores/country"
+  import { PYTHON_API } from "../variables"
+
+  const lookupCountry = async () => {
+    await fetch(`${PYTHON_API}/country`)
+      .then(async (response: Response) => {
+        const data = await response.json()
+
+        if (!response.ok) {
+          alert("Failed to lookup country")
```

Comment:
Alerts are so 2004. Should maybe be done another way as to not confuse the user.

Reply:
Yea that was mostly for debugging - I just need people to test if the thing works. I have a hard time figuring out the corner cases

Reply:
Maybe a modal with some text along the lines of: "We could not automatically detect which country you're in. Use the blahblahblah to choose your country."

Reply:
Alright :^)

Reply:
Yea, a modal could be nice